web/src/beta/components/PublishStateSwitchField/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { FC, useState } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+type Props = {
+  list: string[];
+  selected?: string;
+  onChange?: (label: string) => void;
```

Comment:
Could you add `isActive: boolean` or something like that, and then change `StatusCircle` 's color to green if this added prop is `true` ?